gnomad_qc/v4/resources/meta.py

```diff
@@ -1,35 +1,67 @@
 """Script containing metadata related resources."""
 from gnomad.resources.resource_utils import TableResource, VersionedTableResource
 
+from gnomad_qc.resource_utils import check_resource_existence
 from gnomad_qc.v4.resources.constants import CURRENT_VERSION
 
 
-def _meta_root_path(version: str = CURRENT_VERSION) -> str:
+def _meta_root_path(version: str = CURRENT_VERSION, data_type: str = "exomes") -> str:
     """
     Retrieve the path to the root metadata directory.
 
-    :param version: gnomAD version
-    :return: String representation of the path to the root metadata directory
+    :param version: gnomAD version.
+    :param data_type: Data type ("exomes" or "genomes"). Default is "exomes".
+    :return: String representation of the path to the root metadata directory.
     """
-    return f"gs://gnomad/v{version}/metadata/exomes"
+    return f"gs://gnomad/v{version}/metadata/{data_type}"
 
 
-def meta_tsv_path(version: str = CURRENT_VERSION) -> str:
+def meta_tsv_path(version: str = CURRENT_VERSION, data_type: str = "exomes") -> str:
     """
     Get the path to the finalized sample metadata information after sample QC.
 
-    :param version: gnomAD version
-    :return: String path to the finalized metadata
+    :param version: gnomAD version.
+    :param data_type: Data type ("exomes" or "genomes"). Default is "exomes".
+    :return: String path to the finalized metadata TSV.
     """
-    return f"{_meta_root_path(version)}/gnomad.exomes.v{version}.metadata.tsv.gz"
+    return (
+        f"{_meta_root_path(version, data_type)}/gnomad.{data_type}."
+        f"v{version}.metadata.tsv.gz"
+    )
 
 
-_meta_versions = {
```

Comment:
should we revive this dictionary so that it's easy to tell which metadata versions exist from a quick scan through this code? something like
```
_meta_versions = {
    "exomes": {
        "4.0": TableResource(
            path="gs://gnomad/v4.0/metadata/exomes/gnomad.exomes.v4.0.sample_qc_metadata.ht"
        ),
    },
    "genomes": {
        "4.0": TableResource(
            path="gs://gnomad/v4.0/metadata/genomes/gnomad.genomes.v4.0.sample_qc_metadata.ht"
        ),
    }
}
```
though this wouldn't get used based on the current structure of `meta()`. what do you think?